### ISSUE TITLE
[UI] ui class of repository-summary : top and bottom borders are not aligned

### ISSUE THREAD

**Issue by everhopingandwaiting** [go-gitea/gitea#10754]
- Gitea version (or commit ref):  1.12.0+dev-24-ged5140d23
- Git version: 2.24
- Operating system: docker
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description
 ui class of repository-summary : top and bottom borders are not aligned

suggestion: 
```css

.repository .repository-summary {
    /* height: 48px; */
    overflow: hidden;
```
...
before 
![image](https://user-images.githubusercontent.com/6021724/76928319-16c17780-691c-11ea-99df-ddc731db24cb.png)

after
![image](https://user-images.githubusercontent.com/6021724/76928287-fbef0300-691b-11ea-84cc-4f604ffbbc70.png)

**Comment by everhopingandwaiting** [comment]
if looks good to you , I will give a pr later

**Comment by silverwind** [comment]
Issue is `.repository-summary` is also used on the main repo page where this explicit height is needed to for https://github.com/go-gitea/gitea/issues/10474. I'd suggest renaming the one one the main page to `.repository-summary-with-languages` and move the `48px` style to that name as well.

**Comment by everhopingandwaiting** [comment]
@silverwind Thanks for your advice , I found when remove the 48px from `.repository-summary`, the main repo page seems looks good! 
can I just to remove this ? not add a new css name just like `.repository-summary-with-languages` ?

![image](https://user-images.githubusercontent.com/6021724/77151951-f1766a00-6ad1-11ea-9c44-53bf8978c2cf.png)

**Comment by silverwind** [comment]
Removing it will re-introduce https://github.com/go-gitea/gitea/issues/10474. I will take a look later.

**Comment by everhopingandwaiting** [comment]
@silverwind sorry , I don`t have a full dev environment，
Some code dependencies are not well checked

**Comment by everhopingandwaiting** [comment]
already fixed by #10754,
